x-pack/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/rule_execution_log/client_for_executors/client_interface.ts

```diff
@@ -77,6 +77,11 @@ export interface RuleExecutionContext {
    */
   executionId: string;
 
+  /**
+   * Current revision of the rule being execution (rule.revision)
+   */
+  revision: number;
```

Comment:
Good call on the rename, will change 👍. And I was adding alphabetically, so that's why it ended up top -- the rename will take care of that though 😅. Thanks @banderror!

Reply:
Addressed in [6f6d6f8](https://github.com/elastic/kibana/pull/153716/commits/6f6d6f8a7e20ad3595e4a2ab5feb263f6ba7b4c0) 🎉